eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -81,4 +92,35 @@ public Input get(final String name) throws MalformedURLException {
         return new InputOf(url);
     }
 
+    /**
+     * Accessor.
+     * @return Template
+     */
+    public String getTemplate() {
+        return this.template;
+    }
+
+    /**
+     * Hash of head master.
+     * @return SHA of commit
+     * @throws IOException if fails
+     * @checkstyle NonStaticMethodCheck (20 lines)
+     */
+    private String getSha() throws IOException {
+        String sha = "master";
+        final String link = "https://home.objectionary.com/tags.txt";
+        final InputStream ins = new URL(link).openStream();
+        final Scanner scanner = new Scanner(ins);
+        while (scanner.hasNextLine()) {
+            final String line = scanner.nextLine();
+            final String[] parts = line.split("\t");
+            if (Objects.equals(parts[1], this.tag)) {
+                sha = parts[0];
+                Logger.info(this, "commit sha is %s", sha);
+                break;
+            }
+        }
+        return sha;
```

Comment:
@yasamprom this algorithm will silently return `master` for any branch/tag that doesn't exist in objectionary? I don't think it's a good idea. Instead, we should throw an exception of branch/tag is not found.